packages/application-shell/src/components/public-page-container/public-page-container.js

```diff
@@ -1,31 +1,96 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
 import { FormattedMessage } from 'react-intl';
-import { Spacings, Text } from '@commercetools-frontend/ui-kit';
+import {
+  Text,
+  Spacings,
+  customProperties,
+} from '@commercetools-frontend/ui-kit';
+import LogoBWWhiteSVG from '@commercetools-frontend/assets/images/ct_logo_bw_white.svg';
+import BackgroundImage from '@commercetools-frontend/assets/images/public-background.png';
 import PortalsContainer from '../portals-container';
-import styles from './public-page-container.mod.css';
 import messages from './messages';
 
 const year = new Date().getUTCFullYear();
 
-const PublicPageContainer = props => (
-  <React.Fragment>
-    <PortalsContainer />
-    <div className={styles.container}>
-      <Spacings.Stack scale="m">
-        {props.children}
-        <Spacings.Inline justifyContent="space-between">
-          <a href={`https://commercetools.com/privacy`} target="_blank">
-            <Text.Detail>
-              <FormattedMessage {...messages.privacyPolicy} />
-            </Text.Detail>
-          </a>
-          <Text.Detail>{`${year} © commercetools`}</Text.Detail>
-        </Spacings.Inline>
-      </Spacings.Stack>
-    </div>
-  </React.Fragment>
-);
+const PublicPageContainer = props => {
+  const privacyPolicyLink = (
```

Comment:
This is a bit odd maybe. Maybe make then React components in the same file? `PrivacyPolicyLink` and `TermsOfServiceLink`?

Reply:
Or simply inline them to the message values

Reply:
Yeah this was unnecessary